Patient-MSSC/src/main/java/com/clinics/patient/entity/Patient.java

```diff
@@ -0,0 +1,36 @@
+package com.clinics.patient.entity;
+
+import lombok.Data;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Size;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+@Data
+@Entity
+public class Patient {
```

Comment:
Are you sure it is a good idea to not use Constructors annotation? If I remember correctly JPA needs an empty constructor, @Data implement arg constructor and ... I'm not sure perhaps empty const will despair?